Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -552,6 +553,51 @@
     guides:
     - Robotics
 
+- type: entity
+  id: Biofabricator
+  parent: BaseLathe
+  name: biofabricator
+  description: Produces animal cubes using biomass.
+  components:
+  - type: Sprite
+    sprite: Structures/Machines/biofabricator.rsi
+    snapCardinals: true
+    layers:
+    - state: icon
+      map: ["enum.LatheVisualLayers.IsRunning"]
+    - state: unlit
+      shader: unshaded
+      map: ["enum.PowerDeviceVisualLayers.Powered"]
+    - state: inserting
+      map: ["enum.MaterialStorageVisualLayers.Inserting"]
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
+  - type: Machine
+    board: BiofabricatorMachineCircuitboard
+  - type: MaterialStorage
+    whitelist:
+      tags:
+        - Sheet
+        - RawMaterial
+        - Ingot
+        - Biomass
+  - type: Lathe
+    idleState: icon
+    runningState: building
+    staticRecipes:
+      - MonkeyCube
+      - KoboldCube
+      - CowCube
+      - GoatCube
+      - MothroachCube
+      - MouseCube
+  - type: EmagLatheRecipes
+    emagStaticRecipes:
+      - AbominationCube
+      - SpaceCarpCube
+      - SpaceTickCube
+      - CockroachCube
```

Comment:
yes i know the indentation on these files is absolutely fucked but the lists are over indented by one